changelogs/unreleased/450-jxun

```diff
@@ -0,0 +1 @@
+Add actions that sync images to GCR.
```

Comment:
we don't collect changelogs by adding such a new file. Instead, we suggest developer to fill in release-note in the PR description. On release, we will compile a list of changelogs out of release-notes using [the open source tool](https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md).

I've added a release-note(aka, changelog) to this PR.

Reply:
Can you remove this file?

Reply:
Got it. Will remove